hudi-sync/hudi-sync-common/src/main/java/org/apache/hudi/sync/common/util/SparkDataSourceTableUtils.java

```diff
@@ -98,8 +98,15 @@ public static Map<String, String> getSparkTableProperties(List<String> partition
 
   public static Map<String, String> getSparkSerdeProperties(boolean readAsOptimized, String basePath) {
     Map<String, String> sparkSerdeProperties = new HashMap<>();
-    sparkSerdeProperties.put("path", basePath);
+    Map<String, String> sparkSerdePathProperties = getSparkSerdePathProperties(basePath);
```

Comment:
This change introduces the instantiation of a new Map and is not necessary.

Reply:
resolved.